sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/TicketCache.java

```diff
@@ -0,0 +1,30 @@
+package com.microsoft.appcenter.utils;
+
+import android.support.annotation.VisibleForTesting;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class TicketCache {
```

Comment:
Can't you make all methods static and remove `getInstance`?

Reply:
Tha's not the pattern we usually use.